### ISSUE TITLE
[bug]: 3.2.0 - Latent to Image crashes when using TAESD

### ISSUE THREAD

**Issue by grunblatt-git** [invoke-ai/InvokeAI#4722]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### VRAM

32

### What version did you experience this issue on?

3.2.0

### What happened?

When using TAESD as VAE, the image generation crashes with the following message

> [InvokeAI]::ERROR --> Error while invoking:
'DecoderTiny' object has no attribute 'mid_block'

### Additional context

This seems to happen independently from the main model used. The same error appears when invoking from the main UI and from within workflows.

TAESD was installed via the Model Manager by providing the link `madebyollin/taesd` as suggested in the [Release Notes](https://github.com/invoke-ai/InvokeAI/releases/tag/v3.2.0rc2)

**Comment by zealsprince** [comment]
Seconding this as I'm encountering the same issue. I don't believe it should be used as the VAE though unless I am mistaken. I was excited for this feature but it is not fully implemented as per #4056

**Comment by grunblatt-git** [comment]
Maybe i am just using it wrong, but my understanding from the [TAESD documentation](https://huggingface.co/madebyollin/taesd) was that it is used instead of a usual VAE. InvokeAI also classifies the model as VAE, so using it instead of another VAE seemed intuitive.

> [TAESD](https://github.com/madebyollin/taesd) is very tiny autoencoder which uses the same "latent API" as Stable Diffusion's VAE.

The [Release Notes](https://github.com/invoke-ai/InvokeAI/releases/tag/v3.2.0rc2) mention that TAESD is supported, but as you pointed out using TAESD for live-preview is still work in progress (see  #4056). So i assumed that using it as VAE is the only way TAESD is supported as of v3.2.0.

**Comment by Millu** [comment]
Yep TAESD is used as a VAE - it stands for Tiny **Auto Encoder** for Stable Diffusion

@grunblatt-git I'm not having any issues using it, these are my generation settings: 
<img width="367" alt="Screenshot 2023-09-29 at 10 16 37 AM" src="https://github.com/invoke-ai/InvokeAI/assets/7254508/4b866b96-76d2-453e-a159-f0e81add1a0c">

**Comment by Millu** [comment]
Update - it should work using fp16/half precision 
<img width="389" alt="Screenshot 2023-09-29 at 11 52 18 AM" src="https://github.com/invoke-ai/InvokeAI/assets/7254508/ab08e2b5-b56a-4c5d-9d12-e74e59e292b7">

**Comment by grunblatt-git** [comment]
closing since this seems to work by now with fp32 precision as well
